lib/DocumentInspectorFactory.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Component\DocumentManager;
+
+/**
+ * Document inspector factory for the default DocumentInspector.
+ */
+class DocumentInspectorFactory implements DocumentInspectorFactoryInterface
```

Comment:
It is definitely not needed until now, and I don't want to introduce more changes than required. Makes stuff more complicated than it has to be. And if we merge it like that it is just there, without any additional value.